website_sale_cart_clear/static/src/js/widget.js

```diff
@@ -0,0 +1,19 @@
+odoo.define("website_sale_cart_clear.widget", function (require) {
+    "use strict";
+
+    var websiteSaleWidgets = require("website_sale.website_sale");
+
+    websiteSaleWidgets.websiteSaleCart.include({
+        events: {
```

Comment:
This breaks [the other events](https://github.com/OCA/OCB/blob/164d99e2822ba6437cad064db818e3397376d0e6/addons/website_sale/static/src/js/website_sale.js#L1004-L1006) of this widget. For example, it's not possible to delete a single line from cart, and neither edit billing address on checkout.

New events should be original ones plus the one added in this module.

Reply:
Applying following change it works properly:
```
        events: _.extend(websiteSaleWidgets.websiteSaleCart.prototype.events, {
            "click .js_clear_cart": "_onClickClearCart",
        }),
```

Reply:
Thanks for having a look!
Indeed you are correct, I overestimated `include`.
Since you know what the fix is, can you propose the change? Otherwise I'll fix it when I get back to this

Reply:
Done, I proposed it as a PR to your repository: https://github.com/SirAionTech/e-commerce/pull/1. Thanks!

Reply:
Merged